be/src/storage/rowset/ordinal_page_index.h

```diff
@@ -129,9 +126,9 @@ class OrdinalIndexReader {
     // valid after load
     int _num_pages = 0;
     // _ordinals[i] = first ordinal of the i-th data page,
```

Comment:
Why use std::unique_ptr to replace std::vector can reduce the memory consumption?

Reply:
std::vector<> will use 24 bytes, std::unique_ptr only use 8 bytes.